tests/python/common/ops/test_ops.py

```diff
@@ -405,6 +405,13 @@ def test_gather_mm_idx_b(feat_size, dtype, tol):
     ):
         pytest.skip("BF16 is not supported.")
 
+    if (
+        F._default_context_str == "gpu"
+        and dtype == torch.float16
+        and torch.cuda.get_device_capability() < (7, 0)
+    ):
+        pytest.skip("FP16 is not supported for atomic operations on this GPU.")
```

Comment:
nit: place torch.cuda.get_device_capability() in the log as well?

Reply:
Thanks!